Library/Homebrew/cli/parser.rb

```diff
@@ -122,8 +122,9 @@ def initialize(&block)
         @args = Homebrew::CLI::Args.new
 
         # Filter out Sorbet runtime type checking method calls.
-        @command_name = caller_locations.select { |location| location.path.exclude?("/gems/sorbet-runtime-") }
-                                        .second.label.chomp("_args").tr("_", "-")
+        cmd_location = caller_locations.select { |location| location.path.exclude?("/gems/sorbet-runtime-") }.second
+        @command_name = cmd_location.label.chomp("_args").tr("_", "-")
+        @is_dev_cmd = cmd_location.path.start_with?(Commands::HOMEBREW_DEV_CMD_PATH)
```

Comment:
```suggestion
        @is_dev_cmd = cmd_location.absolute_path.start_with?(Commands::HOMEBREW_DEV_CMD_PATH)
```

It might make more sense to use `absolute_path` here though `path` and `absolute_path` return the same result on my machine. Either way `Commands::HOMEBREW_DEV_CMD_PATH` uses the absolute path.